src/pages/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+
+import { Container, Box, Grid, TextField, Typography }from '@mui/material';
+import { useToast } from '@chakra-ui/react';
+
+import CommonButton from '../../components/Button/CommonButton';
+import { getLoginPageStyles } from './LoginPage.styles';
+import BrandIdentity from '../../components/BrandIdentity/BrandIdentity';
+
+import LoginPageImage from '../../assets/loginPageImage1.gif';
+import { useDispatch } from 'react-redux';
+import { setUserData } from '../../store/slices/UserDataSlice';
+
+const LoginPage = () => {
+
+    const { classes } = getLoginPageStyles();
+    
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const userDetails = {//object to store user login details
```

Comment:
it can be written as 
const userDetails = {//object to store user login details
        email,
        password
    }